### PR TITLE
[feature](merge-cloud) Add cloud meta manager rpc implementation

### PR DESCRIPTION
**Pull request by w41ter**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by w41ter** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: be/src/cloud/cloud_meta_mgr.cpp [hunk 1]
```diff
@@ -241,49 +296,194 @@ Status CloudMetaMgr::update_tmp_rowset(const RowsetMeta& rs_meta) {
 }
 
 Status CloudMetaMgr::commit_txn(StreamLoadContext* ctx, bool is_2pc) {
```
Comment (platoneko) [inline be/src/cloud/cloud_meta_mgr.cpp:298]: 像这种入参不变的签名都用 const & 不？

**Comment by platoneko** [comment]
meta_mgr.h 感觉可以不要了

File: be/src/cloud/cloud_meta_mgr.cpp (hunk 1 above)
Comment (w41ter) [reply be/src/cloud/cloud_meta_mgr.cpp:298]: 可以，我改一下好了

**Comment by w41ter** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by w41ter** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

**Review by hello-stephen — APPROVED** [review]

**Pull request merged**